pkg/config/config.go

```diff
@@ -45,7 +45,8 @@ type config struct {
 var c config
 
 var (
-	EnabledEBPFCgroupID = false
+	EnabledEBPFCgroupID           = false
+	EnabledHardwareCounterMetrics = true
```

Comment:
Again, `EnabledHardwareCounterMetrics` might be misleading
We will never disable the hardware counter metrics if they are available, what we might disable is expose those metrics to prometheus...